modules/sara_r4/cellular_r4_api.c

```diff
@@ -1776,8 +1776,8 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Bit Error Rate (BER):
-             *  � 0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
-             *  � 99: not known or not detectable
+             *  �E0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
```

Comment:
I think this modification is because original comments use special character.
Can you help to update to the following or leave it unchanged, then we fix in another PR?
```
* 0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
```

Reply:
Thanks for review.
I see. I will update this PR.